gef-ui/server/controllers/amendments/cover-end-date/get-cover-end-date.ts

```diff
@@ -0,0 +1,39 @@
+import { CustomExpressRequest } from '@ukef/dtfs2-common';
+import { Response } from 'express';
+import * as api from '../../../services/api';
+import { CoverEndDateViewModel } from '../../../types/view-models/amendments/cover-end-date-view-model';
+import { asLoggedInUserSession } from '../../../utils/express-session';
+import { userCanAmendFacility } from '../../../utils/facility-amendments.helper';
+
+export type GetCoverEndDateRequest = CustomExpressRequest<{
+  params: { dealId: string; facilityId: string; amendmentId: string };
```

Comment:
Should these not be `ObjectID`?

Reply:
@abhi-markan these are req.params values extracted from the URL as strings I think these will always be strings ?

Reply:
Yes of!course please ignore.